fe/src/main/java/org/apache/doris/catalog/Column.java

```diff
@@ -419,6 +485,14 @@ public void write(DataOutput out) throws IOException {
         stats.write(out);
 
         Text.writeString(out, comment);
+
+        if (Catalog.getCurrentCatalogJournalVersion() >= FeMetaVersion.VERSION_74) {
```

Comment:
Since Column is already support GSON serde, we can change the write method to GSON.
And write method does not need to judge the FeMetaVersion

Reply:
OK, we can refactor the serialized and deserialized method of Column in other issuses?